robot_smach_states/examples/example_give_directions.py

```diff
@@ -0,0 +1,22 @@
+import rospy
```

Comment:
Should this file be executable? Currently is not.

Reply:
Decision: all files should be non executable. Done!